cfme/utils/appliance/implementations/rest.py

```diff
@@ -2,6 +2,9 @@
 
 
 class ViaREST(object):
+
+    name = "REST"
```

Comment:
**QUESTION** each Name equals the class name in a way, and the intended result of `__str__`

i wonder why that is needed

Reply:
params getting tried to get the name of an object which is the right way to do it